### DISCUSSION TITLE
Many pages in to character select. You can change the pages pressing tag ikemen new buttons.

### DISCUSSION THREAD

**Discussion by Splashslash** [ikemen-engine/Ikemen-GO#874] [Feature Request]
how have many pages in character select  screen. Exemple: if someone want 500 chars in mugen, 50 char per page. without reduncing portraits or put more than character in a slot. You can  change preesing the same new buttons of tag. This can be a global solution for anyone that wants many characters. And use one more character per slot when is ryu, evil ryu etc

I also really wanted scrollable selection screen

Solves many problems and leaves the player free to add as many pages as they want.

Maybe with groove system update will be possible put Scrollable select screen...

if my memory serves, it will be possible to play  like smash bros, and put more combo announcers.

**Comment by jefinhoyagami** [comment]
Will be a good improvement!
